### PR TITLE
Move font and color definitions into style

### PR DESCRIPTION
**Pull request by shichongrui**

### CODE REVIEW

File: src/apply_styles.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    let argb = parsed.rgba.slice()
+    argb.unshift(Math.round(argb.pop() * 255))
+    return gui.Color.argb(...argb)
+  }
+}
+
+function applyTextStyles (view, styles) {
+  let { fontWeight, fontFamily, fontSize, fontStyle } = styles
+  fontWeight = fontWeight || ''
+  fontWeight = FONT_WEIGHTS[fontWeight.toString()] || fontWeight
```
Comment (oyyd) [inline src/apply_styles.js:40]: Maybe we can log a warning message in the future If users have provided an invalid property.

File: src/apply_styles.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  if (styles.verticalAlign) {
+    view.setVAlign(VALIGN_MAPPING[styles.verticalAlign])
+  }
+}
+
+module.exports = function applyStyles (view, styles = {}) {
+  if (view instanceof gui.Label || view instanceof gui.Entry) {
+    applyTextStyles(view, styles)
```
Comment (oyyd) [inline src/apply_styles.js]: It seems that `applyTextStyles` doesn't apply other properties like `width`, `height` and those from yoga layout.

Comment (shichongrui) [reply src/apply_styles.js]: Ah you are correct. I will fix that.

File: src/apply_styles.js (hunk 1 above)
Comment (shichongrui) [reply src/apply_styles.js:40]: Yeah I think it would be good to inform users of what style properties we don't support yet.

Comment (shichongrui) [reply src/apply_styles.js:40]: Once this is merged I will create an issue so we don't forget about this.

**Pull request merged**